<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGenius</class>
 <widget class="QDialog" name="AddGenius">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelAddGeniusHeading">
     <property name="text">
      <string>&lt;h1&gt;Add a genius&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelGeniusName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGeniusNameError">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditGeniusName">
     <property name="placeholderText">
      <string>Enter genius name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelGenusGender">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGenderError">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxGeniusGender"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelGeniusBirthYear">
       <property name="text">
        <string>Birth Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGeniusBirthYearError">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditBirthYear">
     <property name="placeholderText">
      <string>Enter Birth Year</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelDeathYear">
       <property name="text">
        <string>Death Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGeniusDeathYearError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditDeathYear">
     <property name="placeholderText">
      <string>Enter Death Year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSaveGenius">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save Genius</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
